### DISCUSSION TITLE
Stree info in G2P output

### DISCUSSION THREAD

**Discussion by ease-zh** [speechbrain/speechbrain#2373] [Feature Request]
### 🚀 The feature

The soundchoice-g2p module returns ARPABET without stress, which is very important for TTS usage.

### Solution outline

g2p-en returns ARPABET  with stress and even keeps the punctuation.

**Comment by BenoitWang** [comment]
Hi @ease-zh, for keeping the punctuations there is a `_g2p_keep_punctuations` [function](https://github.com/speechbrain/speechbrain/blob/9ca48f290a7667bd7e0f6f53ec041c814a3a38af/speechbrain/utils/text_to_sequence.py#L323) in `speechbrain.utils.text_to_sequence`, however I don't think we have stress yet. @flexthink may know more about it.
